src/imports/logger/ErrorsTransport.js

```diff
@@ -0,0 +1,77 @@
+//import Rollbar from 'rollbar'
+
+import * as Sentry from '@sentry/node'
+import Transport from 'winston-transport'
+import { SPLAT } from 'triple-beam'
+import { forEach } from 'lodash'
+import Config from '../../server/server.config'
+
+const { env, sentryDSN, version, network } = Config
+const logEnvAllowed = !['test', 'development'].includes(env)
+
+// let rollbar
+// if (logEnvAllowed && rollbarToken) {
+//   rollbar = new Rollbar({
+//     accessToken: rollbarToken,
+//     captureUncaught: true,
+//     captureUnhandledRejections: true,
+//     payload: {
+//       environment: process.env.NODE_ENV
+//     }
+//   })
+// }
+
+let sentryInitialized = false
+if (logEnvAllowed && sentryDSN) {
+  Sentry.init({
+    dsn: sentryDSN,
+    environment: env
+  })
+
+  Sentry.configureScope(scope => {
+    scope.setTag('appVersion', version)
+    scope.setTag('networkUsed', network)
+  })
+
+  sentryInitialized = true
+}
+
+class CustomTransport extends Transport {
```

Comment:
it's ErrorTransport or CustomTransport ? use one name everywhere please
according to it's destination 'SentryTransport' would be more preferrable